SCOPES.md

```diff
@@ -10,9 +10,8 @@ Supported scope prefixes:
   * Used for android Firefox Focus style signing tasks
 
 # Scopes
-* `{scope_prefix}:autograph:{autograph_keyid}`
+* `{scope_prefix}:autograph:dep-signing`
```

Comment:
Hm, doesn't [this code](https://github.com/mozilla-releng/signingscript/pull/53/files#diff-bd999ac053e8b6b2f4d811cb1bc52909R663) mean we're using `{scope_prefix}:cert:dep-signing`? That would suggest `{scope_prefix}:autograph:*` is deprecated.

If so, we can probably also drop the `project:releng:signing:autograph:dep-signing` scope from test_autograph.sh

Reply:
yep!